src/frontend/src/App.tsx

```diff
@@ -23,16 +26,21 @@ function App() {
 
   return (
     <QueryClientProvider client={queryClient}>
+      <AccessibilityHelper />
+      <PerformanceMonitor enabled={process.env.NODE_ENV === 'development'} />
+      <ThemeToggle />
       {!isSDKContext && <AppInitialization />}
       <Suspense fallback={null}>
         <I18nProvider locale={i18n.language}>
           <Layout>
-            <Switch>
-              {Object.entries(routes).map(([, route], i) => (
-                <Route key={i} path={route.path} component={route.Component} />
-              ))}
-              <Route component={NotFoundScreen} />
-            </Switch>
+            <main id="main-content">
```

Comment:
This is certainly a mistake?

From my understanding this would make the whole app wrapped in a main tag, always. The header component exposes header/nav elements, and we already use a main tag in the content components. And, adding this would make the skip links basically useless as they would just skip the whole app that is rendered right after the skip links.